src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -17,48 +17,48 @@
 $event_id = $event->ID;
 $day_number = $this->get( 'day' );
 
-$classes = [ 'tribe-events-calendar-month__event-multiday' ];
+$classes = [ 'tribe-events-calendar-month__multiday-event' ];
 
 // Check if it's featured.
 if ( $is_featured = isset( $event->featured ) && $event->featured ) { // @todo: later use tribe( 'tec.featured_events' )->is_featured( $event_id ) or similar
-	$classes[] = 'tribe-events-calendar-month__event-multiday--featured';
+	$classes[] = 'tribe-events-calendar-month__multiday-event--featured';
```

Comment:
Maybe we can leave a `@todo` for BE devs to be sure that we later move the classes configuration to a template tag, something like what `post_class` is for WP.